### PR TITLE
V2

### PR DESCRIPTION
**Pull request by aStereoID**
I have changed the format rules to finally resolve the years correctly. 
However, I have no way of testing whether this really works.

Correct would be e.g.: 
1601 = šěsnaćestow jedyn
1854 = wosomnaćestow štyriapołsta
1217 = dwanaćestow sydomnaće

This was already an issue: #5 
Also unsolved so far, I guess: #6
Is there a chance to solve both quickly and easily before merge?

### CODE REVIEW

**Comment by aStereoID** [comment]
just also changed test resource [numbers.csv](https://github.com/marytts/marytts-lang-hsb/blob/v2/src/test/resources/marytts/language/hsb/numbers.csv)

File: src/main/resources/marytts/language/hsb/formatRules.txt [hunk 1]
```diff
@@ -174,22 +174,6 @@ x.x: << koma >>;
...
-1500: pjatnaćestow;
-1501-1599: pjatnaćestow[ >>];
-1600: šěsnaćestow;
-1601-1699: šěsnaćestow[ >>];
-1700: sydomnaćestow;
-1701-1799: sydomnaćestow[ >>];
-1800: wosomnaćestow;
 1801-1899: wosomnaćestow[ >>];
 1900: dźewjatnaćestow;
 1901-1999: dźewjatnaćestow[ >>];
```
Comment (psibre) [inline src/main/resources/marytts/language/hsb/formatRules.txt:179]: These three lines at the end still cause the error
```
java.lang.IllegalArgumentException: Illegal character - in rule descriptor
```
for me, but if I remove them, the tests pass.

Comment (aStereoID) [reply src/main/resources/marytts/language/hsb/formatRules.txt]: True, I just deleted them. Thanks for checking!

**Review by aStereoID — COMMENTED** [review]
I approve :-)

**Pull request merged**